server/request.go

```diff
@@ -136,7 +132,7 @@ func (r *RpcRequest) process() {
 	}
 
 	r.log("JSON-RPC method: %s ip: %s", r.jsonReq.Method, r.ip)
-	MetaMaskFix.CleanupStaleEntries()
+	State.cleanup()
```

Comment:
Do we need to do this on every rpc request? Does it matter? Alternative is to only call on eth_sendRawTransaction

Reply:
well spotted, thanks!